test/cli/test/bun-test.test.ts

```diff
@@ -846,6 +846,139 @@ describe("bun test", () => {
     });
     test.todo("check formatting for %p", () => {});
   });
+  describe(".failing", () => {
+    test("expect good fail", () => {
+      const stderr = runTest({
+        args: [],
+        input: [
+          `
+          import { test, expect } from "bun:test";
+          test.failing("the", () => {
```

Comment:
[`test.failing`](https://jestjs.io/docs/api#testfailingname-fn-timeout) is a Jest API, but it does not work like this and it will probably confuse people.

How it works in Jest is that it is expected that the test fails, so the test is actually marked as passed.